cypress/support/index.js

```diff
@@ -1,6 +1,33 @@
+beforeEach(() => {
+  cy.fixture('user_config').as('userConfig');
+  cy.server();
+  cy.route('/user-config/config.json', '@userConfig');
+});
+
 Cypress.Commands.add('login', () => {
-  cy.fixture('realm').then((realm) => {
-    cy.visit(`/auth?realm=${realm.name}#access_token=${realm.infinite_token}`);
-    cy.wait(500);
-  });
+  cy.fixture('user_config')
+    .as('userConfig')
+    .then((userConfig) => {
+      cy.server();
```

Comment:
no need to redefine a server/route here. it's already defined in the previous `beforeEach`